src/style.css

```diff
@@ -1 +1,90 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.logo__moyo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.nav__links {
+  margin: 0;
+  padding: 0;
+  text-transform: uppercase;
+  color: #000;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.link {
+  margin: 0 10px;
+  height: 60px;
+  text-decoration: none;
+  color: inherit;
+  white-space: nowrap;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.first {
+  color: #00acdc;
+  margin: 0 10px 0 0;
+  height: 60px;
+  text-decoration: none;
+  white-space: nowrap;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.last {
+  margin-right: 0;
+}
+
+.is-active.first::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  background: #00acdc;
```

Comment:
move same color to the var

Reply:
sorry. i don't understand. what is var in this case?